src/components/App/App.js

```diff
@@ -14,13 +15,19 @@ export default class App extends Component {
   componentDidMount = () => {
     this.setState({movies: movieData.movies});
   }
+  fetchSingleMovie = (movieId) => {
+    this.setState({selectedMovie: movieData.movie});
```

Comment:
NIce, we just need to remember to update the selected movie (utilizing find? with one of the data sets)